app/models/bot/smooch.rb

```diff
@@ -538,6 +538,10 @@ def self.process_menu_option_value(value, option, message, language, workflow, a
     end
   end
 
+  def self.is_a_shortcut_for_submission?(state, message)
+    self.is_v2? && (state == 'main' || state == 'waiting_for_message') && (!message['mediaUrl'].blank? || ::Bot::Alegre.get_number_of_words(message['text'].to_s) > 3) # FIXME: Confirm this number
```

Comment:
I think it's better to get this number `> 3` from config or settings instead of hard coded

Reply:
Good point, Sawy! Done.